src/SFA.DAS.ApplyService.Database/Script.PostDeployment1.sql

```diff
@@ -133,3 +133,7 @@ BEGIN
 		   (10048569, N'2021-07-01 08:00:00', N'2021-07-31 23:59:59'),
 		   (10055542, N'2021-07-01 08:00:00', N'2021-07-31 23:59:59')
 END
+
+-- APR-2494 script to update GatewayReviewStatus Reject to Rejected
+-- Can be removed once APR-2494 deployed to prod
+  update apply set GatewayReviewStatus='Rejected' where GatewayReviewStatus='Reject'
```

Comment:
Probably didn't need to do this as I don't think any applications have made it into gateway in PROD but good to be proactive I guess